Menu screen

randomPoison opened the pull request:
- Adds a MainMenu scene for starting and ending the game.
- Main menu contains "Play" and "Quit" buttons and has a camera flythrough of the level in the background.

randomPoison committed:
Add Main Menu Scene

- Creates a duplicate of the BlockOut scene for displaying the main menu.
- Adds a main menu interface for starting the game or exiting.

randomPoison committed:
Menu Flythrough

- Adds a camera flythrough for the main menu scene.

brickman1444 commented:
Looks good. The only thing you should add is pull in my changes from the cinematics, then update the LevelUtils list of scene files with the correct name for the main menu scene file.

randomPoison committed:
Merge remote-tracking branch 'origin/master' into menu_screen

Conflicts:
	Assets/Prefabs/UI/UICanvas.prefab

randomPoison committed:
Escape To Menu

- Remove quitting the game by pressing escape, replace with return to menu with escape button.
